### DISCUSSION TITLE
Controlling tab suspension?

### DISCUSSION THREAD

**Discussion by ghbtsm** [sienori/Tab-Session-Manager#1147] [General]
Hi, I've been trying Tab Session Manager but there is one issue that I can't find any solution for.

Recently, Tab Session Manager seems to be automatically suspending tabs. At least, the favicon for the tab changes to a Tab Session Manager icon in the row of tabs at the top of Chrome.

This seems to break OneTab, which doesn't save those suspended tabs.

I can't find any controls for Tab Session Manager to disable the suspension or to change how long to wait before suspending a tab.

Am I missing something? I was planning to donate but I am going to have to move to a different session manager if this can't be fixed.

Thank you.

**Comment by bandaid** [comment]
Yes, we need the ability to disable tab suspension. I believe this ought to be an essential component and obvious to disable, especially for those users with over 64 GB of memory.
